### PR TITLE
Create topical-chat-enriched.yaml

### PR DESCRIPTION
**Pull request by bhedayat**
*Issue #, if available:*

*Description of changes:*
Creating topical-chat-enriched.yaml

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: f9a76fb83e3da60fe5a48d574f0df91bf39a088e
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=0469a15e-7417-4d71-a2b3-0666ef7a29b7%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/topical-chat-enriched.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Name: Enriched Topical-Chat Dataset for Knowledge-Grounded Dialogue Systems
+Description: |
+  This dataset provides extra annotations on topic of the publicly released
```
Comment (g-karthik) [inline datasets/topical-chat-enriched.yaml:3]: on "top"

File: datasets/topical-chat-enriched.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Name: Enriched Topical-Chat Dataset for Knowledge-Grounded Dialogue Systems
+Description: |
+  This dataset provides extra annotations on topic of the publicly released 
+  Topical-Chat dataset(https://github.com/alexa/Topical-Chat) which will help in reproducing our results in our paper
```
Comment (g-karthik) [inline datasets/topical-chat-enriched.yaml:4]: reproducing "the"

File: datasets/topical-chat-enriched.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Name: Enriched Topical-Chat Dataset for Knowledge-Grounded Dialogue Systems
+Description: |
+  This dataset provides extra annotations on topic of the publicly released 
+  Topical-Chat dataset(https://github.com/alexa/Topical-Chat) which will help in reproducing our results in our paper 
+  "Policy-Driven Neural Response Generation for Knowledge-Grounded Dialogue Systems" (https://arxiv.org/abs/2005.12529?context=cs.CL). 
+  The dataset contains 5 files: train.json, valid_freq.json, valid_rare.json, test_freq.json and test_rare.json. 
+  Each of these files will have additional annotations on top of the original Topical-Chat dataset.
+  These specific annotations are: dialogue act annotations and knowledge sentence annotations.
+  The annotations were computed automatically use off the shelf models which are mentioned in the README.txt
```
Comment (g-karthik) [inline datasets/topical-chat-enriched.yaml:9]: automatically "using"

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 90fb2a3cd71dcc007ddf776c6ee66f146cdb9430
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2fb48414-da97-4b5b-9118-294bb3cbf60c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 00beb9a83f4a32a33523510d86fba9348f3545a9
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=ecc5f137-b8c3-448d-90b1-fc452ffc7e2e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**